src/third-party/pinyin/pinyin_data.hpp

```diff
@@ -8,6 +8,7 @@
         extracted kMandarin and kHanyuPinyin field
         converted to a map structure
         removed accents on letter
+        mapped "ü" to both "u" and "v"
```

Comment:
I'm afraid a simple mapping from `ü` to `u` and `v` won't work. There are `ü` sounds that are spelled as `u` in standard pinyin such as `ju` which can also be spelled as `jv` and some pinyin with the `ü` sound cannot be spelled with `u` (for example `lü` cannot be spelled as `lu`)

Reply:
Oh, didn't thought about this.. There are two cases with this problem, lu and nu, it should be fixed now. Thank you for pointing this out